Makefile

```diff
@@ -115,7 +115,7 @@ ifdef REPORT
 endif
 
 codespell:
-	codespell --skip=./.git/*,*.pyc,leapp.db
+	codespell --skip=./.git/*,*.pyc,leapp.db --ignore-words-list=subprocess\' repos
```

Comment:
The codespell workflow should also follow this behaviour imo.

Reply:
Alright makes sense. 

> Are you referring to the github workflow? If yes, I didn't see the need to run it on different versions and since there's a GH Action for it, I just run it

I was thinking whether the codespell gh workflow would not have problems with the not-skipped repos, but seems it doesn't.